src/test/java/org/opensrp/service/multimedia/S3MultimediaFileManagerTest.java

```diff
@@ -141,16 +134,4 @@ public void testGetS3FilePathShouldReturnCorrectFilePath() throws Exception {
 		assertEquals("bucket_folder_path" + File.separator + "path/to/file", truncatedFilePath);
 	}
 
-	@Test
-	public void testUploadShouldUploadFileWithCorrectName() throws Exception {
```

Comment:
Why is this test deleted

Reply:
The original file name and the bytes are passed as constants from server-web. 
The test is verifying that the Multipart file name is similar to the uploaded file name since we can't know the file name from the bytes, we can have this test moved to server web

Reply:
ok, please move this to the web test